guides/common/modules/proc_creating-a-suse-operating-system.adoc

```diff
@@ -1,10 +1,12 @@
 [id="Creating_a_SUSE_Operating_System_{context}"]
 = Creating a SUSE Operating System
 
-{ProvisioningDocURL}adding-installation-media_provisioning[Create an installation media] and {ProvisioningDocURL}creating-operating-systems_provisioning[operating system] entry for SLES 15 SP3.
+Use this procedure to create an installation medium and operating system entry for SLES 15 SP3.
```

Comment:
Do we support newer SP versions?

Reply:
Yes, but introducing attributes for OS major and minor version will happen in a separate PR.